### ISSUE TITLE
Run parallel on specific devices

### ISSUE THREAD

**Issue by akbrkml** [AppiumTestDistribution/AppiumTestDistribution#1085]
Hi @saikrishna321 

I'm trying to run a parallel tests with specific devices. 

Let's say, I have 10 devices online and just wanted to run on 2 devices.

I'm implemented this config `"appium:udids": "device, device2"` nothing happens. My 10 devices still run the parallel tests.

And then trying another config with `"udids": "device, device2"` got this error: `java.lang.IllegalArgumentException: Multiple entries with same key: appium:udids=emulator-5556 and appium:udids=emulator-5554`

Here is the full [logs](https://gist.github.com/akbrkml/2489d16e6113dffaac43675ce0c32f1f).

Thank you.

**Comment by saikrishna321** [comment]
@akbrkml which version of ATD are you using?

**Comment by akbrkml** [comment]
this version a1482f15da

**Comment by akbrkml** [comment]
@saikrishna321 any updates?

**Comment by saikrishna321** [comment]
No! I'm busy with other things. Wont look at it immediately. Happy to review a PR.
